app/src/main/res/layout/fragment_cart.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
We only need a scrollView for the products in the cart, not for all screen

Reply:
because screen is bigger so we must use scroll view even if we use  recycler